app/controllers/shipit/github_authentication_controller.rb

```diff
@@ -20,8 +22,9 @@ def logout
     private
 
     def sign_in_github(auth)
-      user = Shipit.github_api.user(auth[:info][:nickname])
-      User.find_or_create_from_github(user).id
+      user = User.find_or_create_from_github(auth.extra.raw_info)
```

Comment:
Nope, it's the user resource from the GitHub api: https://github.com/intridea/omniauth-github/blob/45f2fc73d6d06f30863adac0e6aa112bcaaadf67/lib/omniauth/strategies/github.rb#L47